app/main/forms/suppliers.py

```diff
@@ -130,46 +130,46 @@ def validate(self):
         if not super(AddCompanyRegistrationNumberForm, self).validate():
             valid = False
         if self.has_companies_house_number.data == "Yes" and not self.companies_house_number.data:
-            self.companies_house_number.errors.append('You must enter a Companies House number.')
+            self.companies_house_number.errors.append('Enter a Companies House number.')
```

Comment:
Change to:

Enter your Companies House number

Reply:
(Decided not to bother as likely deleting)